templates/setenv.sh.erb

```diff
@@ -46,3 +34,62 @@ cd $PUSHED_DIR
 
 echo ""
 echo "Server startup logs are located in $LOGBASEABS/logs/catalina.out"
+
+<%- if scope.lookupvar('confluence::version') =~ /^6/ -%>
```

Comment:
can you replace the scope.lookupvar? This is the old way. Docs: https://puppet.com/docs/puppet/5.3/lang_template_erb.html#accessing-puppet-variables